examples/common/python/shared_kv/remote_lmdb/lmdb_helper_proxy.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from service_client.generic import TextServiceClient
+from string_escape import escape, unescape
+import logging
+
+logger = logging.getLogger(__name__)
+#------------------------------------------------------------------------------
+class LMDBHelperProxy:
+    """
+    LMDBHelperProxy passes commands serialized as strings 
+    to the LMDB remote listener.
+    """
+
+    def __init__(self, uri):
+        self.set_remote_uri(uri)
+
+    def set_remote_uri(self, uri):
+        self.__uri_client = TextServiceClient(uri)
+
+#------------------------------------------------------------------------------
+    def set(self, table, key, value):
+        """
+        Function to set a key-value pair in a lmdb table 
+        Parameters:
+           - table is the name of lmdb table in which 
+             the key-value pair needs to be inserted.
+           - key is the primary key of the table.
+           - value is the value that needs to be inserted in the table.
+        """
+        request = "S\n" + escape(table) + "\n" + escape(key) + \
+            "\n" + escape(value)
```

Comment:
Done, this was decided with Eugene with reasoning that string is more generic than JSON and strings might need to be stored.

Reply:
IMO JSON serialization looks more generic than custom serialization approach. For example, if we want to send multiple commands (to avoid sending multiple commands over network), then using JSON is easier to maintain.